src/counts.h

```diff
@@ -0,0 +1,22 @@
+/// @file
+/// SPDX-License-Identifier: GPL-3.0-or-later
+/// @author Simon Heybrock
+/// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory, NScD Oak Ridge
+/// National Laboratory, and European Spallation Source ERIC.
+#ifndef COUNTS_H
+#define COUNTS_H
+
+#include <initializer_list>
+
+#include "dimension.h"
+
+class Dataset;
+
+namespace counts {
```

Comment:
Alternative would be to have additional namespace `density` and then describe all functions as `to..` or `from..` in each.

Reply:
Not sure `density` here is a concept on its own, it is linked to counts.